ecosystem_tests/ecosystem_tests_cli/commands/blueprint_linting.py

```diff
@@ -0,0 +1,138 @@
+########
+# Copyright (c) 2014-2022 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import json
+import subprocess
+from git import Repo
+from github import Github
+from datetime import datetime
+from ...ecosystem_tests_cli import ecosystem_tests
+
+
+@ecosystem_tests.command(
+        name='blueprint-linting',
+        short_help='validate blueprints in a repo using cfy-lint autofix.')
+@ecosystem_tests.options.access_token
+@ecosystem_tests.options.repo_name
+@ecosystem_tests.options.directory
+@ecosystem_tests.options.pull_request_title
+def blueprint_linting(access_token=None,
+                      repo_name=None,
+                      directory=None,
+                      pull_request_title=None):
+
+    time = str(datetime.now())
+    time = time.replace(' ', '_')
+    time = time.replace(':', '-')
+    time = time.replace('.', '-')
+
+    branch_name = time
+
+    if not access_token:
+        access_token = os.environ("GITHUB_TOKEN")
+    if not repo_name:
+        org_name = os.environ("CIRCLE_PROJECT_USERNAME")
+        repo_name = org_name + "/" + os.environ("CIRCLE_PROJECT_REPONAME")
+    if not directory:
+        directory = "/tmp/" + time
```

Comment:
doing this differs to the default option when not specifying a dir (the one the terminal is on)
i think it might be best to let go of the directory and simply always doing

`directory = tempfile.mkdtemp(prefix=time)`